### PR TITLE
Add skiplink component

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Add skiplink as a Tailwind component (Note: I increase the font size a bit since I think it's a bit too small in Decanter 6).
- Also note: I did `npm run publish` and `npm run wind` this time so perhaps that's why there are so many new lines added. I only did `npm run build` in previous PRs. This seems to fix the issue you saw where the decanter-utilities.css file was bigger than the main decanter.css file.

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Go to preview build.
1. When the page finish loading, hit tab with the keyboard - check that the skip link button show up.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
> * Also note: I did `npm run publish` and `npm run wind` this time so perhaps that's why there are so many new lines added. I only did `npm run build` in previous PRs. This seems to fix the issue you saw where the decanter-utilities.css file was bigger than the main decanter.css file.

https://github.com/SU-SWS/decanter/blob/ADAPT-2059-skiplink/package.json#L12

the wind command only compiles the one file. The build/publish commands should compile all the files. We have been using the build command as it is pretty code and easy to read while we lack documentation. I want to eventually move to `npm run publish` but we need more documentation first.

**Comment by yvonnetangsu** [comment]
> > * Also note: I did `npm run publish` and `npm run wind` this time so perhaps that's why there are so many new lines added. I only did `npm run build` in previous PRs. This seems to fix the issue you saw where the decanter-utilities.css file was bigger than the main decanter.css file.
> 
> https://github.com/SU-SWS/decanter/blob/ADAPT-2059-skiplink/package.json#L12
> 
> the wind command only compiles the one file. The build/publish commands should compile all the files. We have been using the build command as it is pretty code and easy to read while we lack documentation. I want to eventually move to `npm run publish` but we need more documentation first.

That sounds good. I just merged the new changes in and going to run build again to resolve the conflict.

File: dist/decanter-base.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
```
Comment (sherakama) [inline dist/decanter-base.js:1]: We don't plan on using these javascript files. I suppose we should add them to the gitignore file. Please remove them from this PR.

**Review by sherakama — APPROVED** [review]
Delete the JS files in the `dist` directory and add them to the .gitignore file then this is GTG.

File: dist/decanter-base.js (hunk 1 above)
Comment (yvonnetangsu) [reply dist/decanter-base.js]: Removed and added dist/*.js to gitignore

**Pull request merged**